py/picca/test/test_1_deltas.py

```diff
@@ -0,0 +1,427 @@
+        cmd = " picca_deltas.py"
+        cmd += " --in-dir " + self._branchFiles + "/Products/Spectra/"
+        cmd += " --drq " + self._branchFiles + "/Products/cat.fits"
+        cmd += " --out-dir " + self._branchFiles + "/Products/Delta_LYA/Delta/"
+        cmd += " --iter-out-prefix " + self._branchFiles + \
+            "/Products/Delta_LYA/Log/delta_attributes"
+        cmd += " --log " + self._branchFiles + "/Products/Delta_LYA/Log/input.log"
+        cmd += " --nproc 1"
+        subprocess.call(cmd, shell=True)
+
+        ### Test
+        if self._test:
+            path1 = self._masterFiles + "/test_delta/delta_attributes.fits.gz"
+            path2 = self._branchFiles + "/Products/Delta_LYA/Log/delta_attributes.fits.gz"
+            self.compare_fits(path1, path2, "picca_deltas.py")
+
+        return
+
+
+
+    def send_delta_Pk1D_minisv(self):
+        """
+            Test of picca_deltas currently only to check if miniSV I/O routines
+            work correctly. The data for this routine is randomly generated as for "send_delta", but
+            in DESI-miniSV format instead of an eBOSS format
+        """
+
+        userprint("\n")
+        ### Path
+        path_to_etc = self.picca_base + '/etc/'
+        ### Send
+        cmd = " picca_deltas.py"
+        cmd += " --in-dir " + self._branchFiles + "/Products/Spectra_MiniSV/"
+        cmd += " --drq " + self._branchFiles + "/Products/cat_minisv.fits"
+        cmd += " --out-dir " + self._branchFiles + "/Products/Delta_Pk1D_MiniSV/Delta/"
+        cmd += " --iter-out-prefix " + self._branchFiles + \
+            "/Products/Delta_Pk1D_MiniSV/Log/delta_attributes"
+        cmd += " --log " + self._branchFiles + "/Products/Delta_Pk1D_MiniSV/Log/input.log"
+        cmd += " --delta-format Pk1D --mode desiminisv --order 0 --use-constant-weight"
+        cmd += " --rebin 1 --lambda-min 3650. --lambda-max 7200.0 --lambda-rest-min 1050.0 --lambda-rest-max 1180"
+        cmd += " --nproc 1"
+        cmd += " --best-obs"
+        cmd += " --mask-file " + path_to_etc + "/list_veto_line_Pk1D.txt"
+        returncode = subprocess.call(cmd, shell=True)
+        self.assertEqual(returncode, 0, "delta_Pk1D_minisv did not finish")
+
+        ### Test
+        if self._test:
+            path1 = self._masterFiles + "/test_delta/delta_attributes_Pk1D_MiniSV.fits.gz"
+            path2 = self._branchFiles + "/Products/Delta_Pk1D_MiniSV/Log/delta_attributes.fits.gz"
+            self.compare_fits(path1, path2, "picca_deltas.py")
+
+            #this checks if any of the output delta files changed
+            for fname in glob.glob(
+                    f'{self._branchFiles}/Products/Delta_Pk1D_MiniSV/Delta/delta-*.fits.gz'
+            ):
+                path2 = fname
+                path1 = f"{self._masterFiles}/test_delta/Delta_Pk1D_MiniSV/{os.path.basename(fname)}"
+                self.compare_fits(path1, path2, "picca_deltas.py")
+        return
+
+
+    def test_delta_Pk1D(self):
```

Comment:
ordering of the methods is weird. Could we put all the test methods together?

Reply:
yes, I didn't clean this up yet, thanks!